source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/add/user-input.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { IAttributeProps } from '@lib/dom';
+
+export interface IProps {
+  onSubmit: (email: string) => void;
+  placeholder?: string;
+}
+
+export interface IState {
```

Comment:
Do we need to export the state interface and IProps?

Reply:
it doesn't but confuses the vscode and you see 100 IProvidedProps as recommendations :(